lib/code_corps/helpers/cloudinary_url.ex

```diff
@@ -0,0 +1,11 @@
+defmodule CodeCorps.Helpers.CloudinaryUrl do
+
+  @host Application.get_env(:arc, :asset_host)
```

Comment:
It will go away. You can comment in #673, too. Can't remove all that until after this is merged into prod.